RELEASE.txt

```diff
@@ -10,42 +10,41 @@ This is the procedure for generating a new Dafny release.
    Edit it into Source/version.cs
    Commit and push this change into the master branch
 
-3) Run 'package.py "$VER"' (in a Linux or Mac system)
+3) Run 'Scripts/package.py "$VER"' (in a Linux or Mac system)
    Warnings about missing pdb files can be ignored.
 
-4) Commit and push the new version of DafnyRef.pdf and possibly Options.txt
-   (.../dafny/docs/DafnyRef/out/DafnyRef.pdf .../dafny/docs/_includes/Options.txt)
-   This version of DafnyRef.pdf is also included in the release zip files.
+4) Run 'Scripts/publish-release.sh $VER' to create the github release
+   and upload files to it. You will need to have a Github Personal
+   Authorization token stored in ~/github.token.
 
-5) Create a new release: On 'https://github.com/dafny-lang/dafny/releases'
-   click 'Draft a new release'
+5) In the Github releases, edit the new release to add in the release notes.
 
-6) Fill in a tag ('v' followed by release name), the release name,
-the release notes. For example:
-   release descriptor: 3.0.0-alpha
-   release tag:        v3.0.0-alpha
-   release name:       Dafny 3.0.0 alpha
+6) Push the Publish button.
 
-7) Add all the .zip files created by step 3 in the Package folder.
+7) Commit and merge the newly created DafnyRef.pdf
+   (in dafny/docs/DafnyRef.out) and Options.txt files. The creating of a
+   PR for this merge will cause the CI tests to be run, which includes
+   downloading the most recent release (this one, once it is Published)
+   and running quick tests.
 
-8) Publish the release (by clicking the button at the bottom of the page)
+8) Update the brew formula for dafny (see below)
 
-9) Update the brew formula for dafny (see below)
-
-10) Announce the new release to the world.
+9) Announce the new release to the world.
 
 
 --------------------------
 
 brew is a package manager for the Mac.
```

Comment:
Out of scope for this PR, but we can consider distributing Dafny as a dotnet tool instead of as a brew package, to reduce our maintenance.

For our customers this would mean that they can install Dafny by installing dotnet and then running `dotnet tool install dafny`, instead of installing brew and then doing `brew install dafny`.

For us it would mean we don't need to think about brew and publishing Dafny as a tool can be an automated part of our release process. I'm guessing we can fully automate the brew part as well, but since we're already using dotnet, using that to distribute the binary would mean fewer moving parts on our end.

Reply:
To be discussed later when the time comes I guess. For now, brew is commonly used by Mac users, though a downloadable standalone executable would generally do as well.